src/yunohost/regenconf.py

```diff
@@ -48,7 +48,7 @@
 @is_unit_operation([("names", "configuration")])
 def regen_conf(
     operation_logger,
-    names=[],
+    names=None,
```

Comment:
More information about default value mutable:
https://stackoverflow.com/a/41686973

Reply:
>Default argument values are evaluated only once at function definition time, which means that modifying the default value of the argument will affect all subsequent calls of the function.

My god that's terrifying o.o